ansible_runner/streaming.py

```diff
@@ -60,12 +64,22 @@ def run(self):
         return self.status, self.rc
 
 
-class Worker(object):
-    def __init__(self, _input=None, _output=None, **kwargs):
+class Worker:
+    def __init__(self, _input=None, _output=None, keepalive_seconds: float | None = None, **kwargs):
         if _input is None:
             _input = sys.stdin.buffer
         if _output is None:
             _output = sys.stdout.buffer
+
+        if keepalive_seconds is None:  # if we didn't get an explicit int value, fall back to envvar
+            # FIXME: emit/log a warning and silently continue if this value won't parse
+            keepalive_seconds = float(os.environ.get('ANSIBLE_RUNNER_KEEPALIVE_SECONDS', 0))
```

Comment:
This is kind of a painful FIXME. It's not that I want logging (how do you even ship the logs back??), but just a simple `string_to_bool` kind of method. Could we just copy the module here into the ansible-runner utils dir?

https://github.com/ansible/ansible/blob/devel/lib/ansible/module_utils/parsing/convert_bool.py

Reply:
I'm not seeing how the `convert_bool` stuff applies here- we always want a number...

Reply:
(I internally changed it to a `float` to make the "way-subsecond acid test" easier, but we always want an actual number)

Reply:
aah, sorry. I get it. So a bad value will probably just error, and I think I'm okay with that.